README.md

```diff
@@ -190,7 +191,8 @@ For more information on Android development, see the Android developer site at:
 
 ## Talk to Us
 
-Visit our GitHub [Issues](https://github.com/aws-amplify/aws-sdk-android/issues) to leave feedback and to connect with other users of the SDK.
+* Visit our GitHub [Issues](https://github.com/aws-amplify/aws-sdk-android/issues) to leave feedback and to connect with other users of the SDK.
```

Comment:
Inconsistent punctuation across these two bullets.

Reply:
Is it to leave feedback and connect with other users? This is a strange bullet. I think we need more prescriptive guidance on when to open an issue (bugs, problems) versus when to go to Discord (to connect with... etc).

Reply:
These are good points. For now I'll add the badge at the top and once we have more clarity on GitHub Issues vs Discussions vs Discord we can add links here.